pypeit/par/util.py

```diff
@@ -678,7 +678,7 @@ def make_pypeit_file(pypeit_file, spectrograph, data_files, cfg_lines=None, setu
 
     # Here we go
     with open(pypeit_file, 'w') as f:
-        f.write('# Auto-generated PypeIt file\n')
+        f.write('# Auto-generated PypeIt file using PypeIt version: {}\n'.format(__version__))
```

Comment:
Nice!

Reply:
Please link to the Open Issue on this and then close it eventually.

Reply:
Done and will do.

Reply:
Thanks for doing this!!